lib/pdk/util/template_uri.rb

```diff
@@ -5,6 +5,15 @@ module Util
     class TemplateURI
       SCP_PATTERN = %r{\A(?!\w+://)(?:(?<user>.+?)@)?(?<host>[^:/]+):(?<path>.+)\z}
 
+      PACKAGED_TEMPLATE_KEYWORD = 'pdk-default'.freeze
+      DEPRECATED_TEMPLATE_URL = 'https://github.com/puppetlabs/pdk-module-template'.freeze
+
+      PACKAGED_TEMPLATE_PATHS = {
+        'windows' => 'file:///C:/Program Files/Puppet Labs/DevelopmentKit/share/cache/pdk-templates.git',
+        'macos'   => 'file:///opt/puppetlabs/pdk/share/cache/pdk-templates.git',
+        'linux'   => 'file:///opt/puppetlabs/pdk/share/cache/pdk-templates.git',
+      }.freeze
```

Comment:
Would it be possible to compute these from the relative path of the current file? I'm thinking about our hope to someday have the packages be relocatable and/or distributing a `.zip` package.

Reply:
Oh, I think this is what Bryan and David were already talking about, this is only for recognizing the package template paths before this change goes out since afterwards this will get converted to a keyword in the metadata. Does that sound correct?

If so, maybe we just rename this const to like `LEGACY_PACKAGED_TEMPLATE_URLS` or something similar?